tests/meta-mender-ci/recipes-testing/read-only-rootfs/mender_%.bbappend

```diff
@@ -0,0 +1,9 @@
+FILES_${PN} += "/data/etc/mender"
+# Run after all mender recipes are finished.
+do_install_append() {
+    mkdir -p ${D}/data/etc/mender
```

Comment:
Is the intention to move this to core or demo layer as well? If that is the case I have two suggestions :).

There is a `IMAGE_FEATURE` called `read-only-rootfs` which we could use to do this conditionally, meaning that this could actually reside in meta-mender-core and applied based on if that feature is set. Are you using the `read-only-rootfs` feature when building a read-only rootfs image?

Secondly, we have `/data/mender` directory where all mender related stuff is stored. I would suggest that these files also go there to make it easy to separate what files are Mender related and what is user data.

Reply:
No, this is meant to be used only for tests. The reason it's needed is that the tests need to make modifications to `/etc/mender/mender.conf`, but users are expected to do this via an update. So making this generic is not of great concern here, since we will be the only users of it.